.circleci/config.yml

```diff
@@ -2,16 +2,35 @@ version: 2.1
 
 aliases:
   - &restore-yarn-cache
-    keys:
-      - v2-yarn-cache
+    key: v2-yarn-cache
+
+  - &restore-node-modules-cache
+    key: v2-node-modules-cache
 
-  # node_modules shouldn't be cached.
-  # See https://github.com/yarnpkg/berry/issues/988
   - &save-yarn-cache
     paths:
       - .yarn/cache
     key: v2-yarn-cache
 
+  - &save-node-modules-cache
+    # Cache big node_modules folders
```

Comment:
(😢We are cursed by not using PnP)

Reply:
Actually, after https://github.com/yarnpkg/berry/pull/955 we don't need to cache it anymore!

Reply:
That PR is merged, guess we can get rid of the cache?